pytest_relative_order.py

```diff
@@ -89,12 +94,12 @@ def sort_DAG(self):
         for nodeid in edges.keys():
             assert not edges[nodeid], f"cycle detected! {edges}"
 
-        return [self.nodeid_to_item[order_id] for order_id in L]
+        return [item for items in [self.nodeid_to_items[order_id] for order_id in L] for item in items]
```

Comment:
This line reminds me... `volume is mounted or not is not mounted` famous comment...